src/validator/test/QueryValidatorTest.cpp

```diff
@@ -229,6 +229,22 @@ TEST_F(QueryValidatorTest, GoWithPipe) {
         };
         EXPECT_TRUE(checkResult(query, expected));
     }
+    {
+        std::string query = "GO 1 STEPS FROM \"1\" OVER like YIELD like._dst AS "
+                            "id | GO 1 STEPS FROM $-.id OVER like YIELD $-.id";
```

Comment:
The second sentence will call buildProjectSrcEdgePropsForGN?

Reply:
yes，the second go sentence will call this func，and it will cause graphd to crash
`if (!exprProps_.inputProps().empty() || !exprProps_.varProps().empty() ||
        !exprProps_.dstTagProps().empty() || from_.fromType != FromType::kInstantExpr) {
        projectSrcEdgeProps = buildProjectSrcEdgePropsForGN(gn->outputVar(), gn);
    }`

Reply:
Add test case in CI makes sense than comparing plan.

Reply:
The plan test case could remain.

Reply:
> The plan test case could remain.

There are already similar test cases in plan test